### PR TITLE
Expose the MetricsFactory

### PR DESCRIPTION
**Pull request by cchacin**
Simple getter to access to the MetricsFactory object.

### CODE REVIEW

File: codahale-extra/src/main/java/com/arpnetworking/metrics/codahale/MetricRegistry.java [hunk 1]
```diff
@@ -159,6 +160,12 @@ public Meter meter(final String name) {
     // TODO(barp): Configurable closer period [#3]
     private static final int CLOSER_PERIOD = 500;
 
+    private final MetricsFactory _metricsFactory;
+
+    public MetricsFactory getMetricsFactory() {
```
Comment (BrandonArp) [inline codahale-extra/src/main/java/com/arpnetworking/metrics/codahale/MetricRegistry.java]: Public methods before private. Move this to line 125 or so.

File: codahale-extra/src/main/java/com/arpnetworking/metrics/codahale/MetricRegistry.java [hunk 2]
```diff
@@ -159,6 +160,12 @@ public Meter meter(final String name) {
     // TODO(barp): Configurable closer period [#3]
     private static final int CLOSER_PERIOD = 500;
 
+    private final MetricsFactory _metricsFactory;
```
Comment (BrandonArp) [inline codahale-extra/src/main/java/com/arpnetworking/metrics/codahale/MetricRegistry.java]: Non-static before static variables.

**Comment by BrandonArp** [comment]
2 comments on style, then you're good to go.

**Comment by cchacin** [comment]
@BrandonArp done

**Comment by BrandonArp** [comment]
Perfect! Thanks for the contribution.  I'll release this ASAP.

**Pull request merged**